terraform/digitalocean/main.tf

```diff
@@ -82,6 +83,30 @@ module "admin_cert" {
   ca_private_key_pem = "${module.ca.ca_private_key_pem}"
 }
 
+module "docker_daemon_certs" {
+  source                = "github.com/Capgemini/tf_tls//docker/daemon"
```

Comment:
I like to have ability and granularity to configure both daemon and client by separate, otherwise you'll be assuming ip_addresses_list or count is same for both which it isn't necessarily